### PR TITLE
Remove native libraries

### PR DESCRIPTION
**Pull request by LeNitrous**
We'll be using a different method of obtaining the dependencies. See https://github.com/vignette-project/osu-framework-live2d/pull/42.

### CODE REVIEW

**Review by sr229 — APPROVED** [review]

**Review by sr229 — COMMENTED** [review]
You'll need to have the new fetching mechanism here then I can merge it to prevent CI failures.

File: .github/workflows/deploy-stable.yml [hunk 1]
```diff
@@ -1,70 +1,63 @@
...
+
+  alert:
+    needs: [ deploy ]
+    runs-on: ubuntu-latest
+      - name: Alert
         uses: tsickert/discord-webhook@v1.0.1
         with:
           webhook-url: ${{secrets.DISCORD_WEBHOOK_URL_STABLE}}
-          content: "@everyone ${{github.ref}} has released. Check https://g.asatomi.live/short/vignette-beta"
+          content: "Hey @everyone! This is your local Insider messenger! This is a test message for Discord-based alerts. If you see something like this in the near future, check the Appcenter!"
```
Comment (sr229) [inline .github/workflows/deploy-stable.yml:63]: Looks like this was unintentionally reverted.

**Review by sr229 — CHANGES_REQUESTED** [review]

**Comment by sr229** [comment]
Closing since it's inactive, feel free to reopen when you get to it again.

**Comment by LeNitrous** [comment]
@sr229 just need this one out and we can get started with the UX rewrite

**Pull request merged**